src/datatypes/src/schema/column_schema.rs

```diff
@@ -43,6 +44,34 @@ pub struct ColumnSchema {
     metadata: Metadata,
 }
 
+impl fmt::Debug for ColumnSchema {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "{} {} {}",
+            self.name,
+            self.data_type,
+            if self.is_nullable {
+                "nullable"
```

Comment:
nitpick: "null" is enough
```suggestion
                "null"
```